SseServer/src/SseServer/Controllers/SseController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Backbone.BuildingBlocks.API;
+using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.Modules.Devices.Application.PushNotifications.Commands.DeleteDeviceRegistration;
+using Backbone.Modules.Devices.Application.PushNotifications.Commands.UpdateDeviceRegistration;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backbone.SseServer.Controllers;
+
+public class SseController : ControllerBase
+{
+    private readonly IEventQueue _eventQueue;
+    private readonly IUserContext _userContext;
+    private readonly IMediator _mediator;
+    private readonly ILogger<SseController> _logger;
+
+    public SseController(IEventQueue eventQueue, IUserContext userContext, IMediator mediator, ILogger<SseController> logger)
+    {
+        _eventQueue = eventQueue;
+        _userContext = userContext;
+        _mediator = mediator;
+        _logger = logger;
+    }
+
+    [HttpGet("/api/v1/sse")]
```

Comment:
> Considering the unsafe nature of this controller

What do you mean with "unsafe"?

> shouldn't we have it behind a POST instead?

1. Semantically it's just a GET that streams data instead of returning it as one thing. So I don't see a reason for a POST.
2. Technically it's meant to be a POST. At least the "offical" `EventSource` library, which is used by JS clients to consume server sent events, only supports GET. 

> Also, why so much logic in its body?

Yeeahh, I'm not 100% happy with this. But on the other hand it's 95% utility code responsible for communication with the client. So exactly what a controller is there for.

Reply:
[Unsafe as defined in RFC9110](https://datatracker.ietf.org/doc/html/rfc9110#name-safe-methods). I mentioned it is not safe because it runs a command (instead of a query).

Thinking a bit more about it, considering it is idempotent (at least as far as I can see), it should actually be a PUT.

Reply:
> it is idempotent (at least as far as I can see)

Ahh, now I understand what you were getting at. But yes, you're right. It is idempotent. 👍